<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>440</height>
    </size>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>560</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>560</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>560</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="select_button">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upgrade_local_button">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>升级底层程序</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upgrade_button">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>升级应用程序</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="textBrowser"/>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>560</width>
       <height>40</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_CAN">
    <property name="title">
     <string>打开CAN设备</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Hex">
    <property name="title">
     <string>Hex设置</string>
    </property>
   </widget>
   <addaction name="menu_CAN"/>
   <addaction name="menu_Hex"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>创芯科技</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
